s/setups/setup-window-events.ts

```diff
@@ -0,0 +1,24 @@
+
+import {StickControls, StickStarters} from "../elements/stick/types.js"
+import {attachEvents} from "../tools/attach-events.js"
+
+export function setupWindowEvents({
+		setTrackingPointerId,
+		getTrackingPointerId,
+		moveStick,
+		resetStick,
+	}: StickStarters & StickControls) {
+
+	return () => attachEvents(window, {
+		pointerup() {
+			setTrackingPointerId(undefined)
+			resetStick()
+		},
+		pointermove(event) {
+			const {clientX, clientY} = <PointerEvent>event
+			const trackingPointer = getTrackingPointerId()
+			if (trackingPointer)
+				moveStick(clientX, clientY)
+		},
+	})
```

Comment:
this is a good example of proper `attachEvents` usage, by returning the disposer back to the `use.setup`